src/index.js

```diff
@@ -46,6 +47,7 @@ class KadDHT extends EventEmitter {
     options = options || {}
     options.validators = options.validators || {}
     options.selectors = options.selectors || {}
+    options.randomWalk = defaultsDeep(options.randomWalk, c.defaultRandomWalk)
```

Comment:
Add `options.randomWalk` to the js docs above and describe the options type for it as follows:

```js
/**
 * Random walk options
 * @typedef {Object} randomWalk
 * @property {boolean} enabled - discovery enabled (default: true)
 * @property {number} queriesPerPeriod - how many queries to run per period (default: 1)
 * @property {number} interval - how often to run the the random-walk process, in milliseconds (default: 300000)
 * @property {number} timeout - how long to wait for the the random-walk query to run, in milliseconds (default: 10000)
*/
```

Also remove `options.enabledDiscovery` from the docs